conbench/tests/api/test_runs.py

```diff
@@ -231,6 +231,27 @@ def test_closest_commit_different_machines_should_not_prefer_test_runs_as_baseli
         response = client.get(f"/api/runs/{contender_run.id}/")
         self.assert_200_ok(response, _expected_entity(contender_run, baseline_run.id))
 
+    def test_baseline_run_always_on_default_branch(self, client):
+        self.authenticate(client)
+        _, benchmark_results = _fixtures.gen_fake_data()
+
+        successes = 0
+        for benchmark_result in benchmark_results:
+            response = client.get(f"/api/runs/{benchmark_result.run_id}/")
+            assert response.status_code == 200, response.status_code
+            baseline_run_link = response.json["links"].get("baseline")
+            if baseline_run_link:
+                baseline_run_response = client.get(
+                    baseline_run_link.replace("http://localhost", "")
+                )
+                assert (
+                    baseline_run_response.status_code == 200
+                ), baseline_run_response.status_code
+                assert baseline_run_response.json["commit"]["branch"] == "default"
```

Comment:
This test does fail on this line on `main`.

Reply:
In the "fake data" the branch is literally called "default"?

Reply:
Yep! https://github.com/conbench/conbench/blob/61fac4e5c12ee4b71a493929662e580084e4186b/conbench/tests/api/_fixtures.py#L287